--- ptlib-2.10.2/ptlib.pc.in.old	2014-03-18 02:21:43.388425144 +0100
+++ ptlib-2.10.2/ptlib.pc.in	2014-03-18 02:19:23.094878006 +0100
@@ -6,6 +6,7 @@
 suffix=
 machtype=@MACHTYPE@
 ostype=@OSTYPE@
+osrelease=@OSRELEASE@
 objdir=obj_@OSTYPE@_@MACHTYPE@${suffix}
 
 Name: ptlib
--- ptlib-2.10.2/make/ptlib-config.in.old	2014-03-18 02:12:44.045163430 +0100
+++ ptlib-2.10.2/make/ptlib-config.in	2014-03-18 02:21:03.497553973 +0100
@@ -4,18 +4,18 @@
 # apparently shamelessly ripped from gtk's gtk-config.in
 #
 
-prefix=@prefix@
-exec_prefix=@exec_prefix@
+prefix=`pkg-config --variable prefix ptlib`
+exec_prefix=`pkg-config --variable exec-prefix ptlib`
 
-VERSION="@PTLIB_VERSION@"
-CCFLAGS="@STDCCFLAGS@"
-LDFLAGS="@LDFLAGS@"
-LIBS="@ENDLDLIBS@ -lpt"
-LIBDIR="@LIBDIR@"
-PTINSTDIR="@prefix@/share/ptlib"
-MACHTYPE="@MACHTYPE@"
-OSTYPE="@OSTYPE@"
-OSRELEASE="@OSRELEASE@"
+VERSION=`pkg-config --modversion ptlib`
+CCFLAGS=`pkg-config --cflags ptlib`
+LDFLAGS=`pkg-config --libs-only-other ptlib`
+LIBS=`pkg-config --static --libs-only-l ptlib`
+LIBDIR=`pkg-config --variable libdir ptlib`
+PTINSTDIR="$prefix/share/ptlib"
+MACHTYPE=`pkg-config --variable machtype ptlib`
+OSTYPE=`pkg-config --variable ostype ptlib`
+OSRELEASE=`pkg-config --variable osrelease ptlib`
 SHAREDLIBEXT="@SHAREDLIBEXT@"
 
 # usage="Usage: ptlib-config [--prefix[=DIR]] [--exec-prefix[=DIR]] [--version] [--libs] [--cflags]"
